pom.xml

```
@@ -3,23 +3,27 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.2</version>
+    <version>4.16</version>
+    <relativePath/>
   </parent>
+  <artifactId>gitlab-plugin</artifactId>
+  <version>${revision}${changelist}</version>
+  <name>GitLab Plugin</name>
+  <url>https://wiki.jenkins-ci.org/display/JENKINS/GitLab+Plugin</url>
```

The plugin documentation in the repository is very, very good.  Let's use it on the plugins.jenkins.io site as well.  See the [documentation as code blog post](https://www.jenkins.io/blog/2019/10/21/plugin-docs-on-github/) and [Using GitHub as a source of documentation](https://www.jenkins.io/doc/developer/publishing/documentation/#documenting-plugins).
```suggestion
  <url>https://github.com/jenkinsci/gitlab-plugin/</url>
```